hack/util-patch-spi-config.sh

```diff
@@ -1,4 +1,4 @@
-#!/bin/bash
+#!/bin/bash -e
```

Comment:
let's keep the scripts consistent - either use this one or the `set -e` one

Reply:
updated